setup.py

```diff
@@ -49,7 +49,7 @@
             'flake8',
         ],
         coverage=[
-            'pytest-cov',
+            'coverage',
```

Comment:
Why are we getting rid of pytest-cov? I think pytest-cov has features that exploit pytest, right? Not that we use distributed testing...